Automattic/wp-calypso
# Woo Tailored Onboarding: Add intro/login and store profiler steps

#### Proposed Changes

* This PR creates the first two steps for the tailored Woo onboarding flow, namely the intro screen (and subsequent login/signup for a logged out user), followed by the store profiler screen. The latter does not have any persistence logic yet, but does populate all necessary dropdowns.

#### Testing Instructions



* Check out this branch locally.
* As a logged-in user, navigate to `/setup?flow=ecommerce` (note that this is a change from the previous name of the flow, which was `tailored-ecommerce`)
* Verify that you see a welcome screen that appears correctly both on mobile and desktop views:

<img width="1164" alt="CleanShot 2022-10-31 at 15 07 10@2x" src="https://user-images.githubusercontent.com/13437011/199100797-309e7ab2-e33a-453b-989c-e12b65ebbef0.png">
<img width="386" alt="image" src="https://user-images.githubusercontent.com/13437011/199100853-e83fc4ce-90b3-4915-a6a0-043ffc3a50f4.png">

* Click "Create Your Store" and verify you are taken to a correctly displayed store profiler form:
<img width="1225" alt="image" src="https://user-images.githubusercontent.com/13437011/199101258-404d4c05-91fc-4313-9627-a3cf6482302e.png">

* Log out of Calypso and go back to `/setup?flow=ecommerce`
* Verify that as a logged out user, clicking "Create Your Store" navigates you to log in or sign up, after which you are redirected back to the store profiler as expected.


#### Pre-merge Checklist






Related to #69386
